uncertainties/core.py

```diff
@@ -1004,6 +1006,76 @@ def ufloat_fromstr(representation, tag=None):
     (nom, std) = str_to_number_with_uncert(representation.strip())
     return ufloat(nom, std, tag)
 
+def ufloat_from_sample(sample, method="gaussian", axis=None):
+    '''
+    Converts a collection of values into a ufloat.
+
+    Arguments:
+    ----------
+    sample: list or numpy array of numbers
+        The sample of values
+
+    method: optional string
+        The method used to calculate the ufloat. currently, only 
+        the 'gaussian' method is implemented.
+
+        gaussian: The nominal value is the mean of the sample. 
+        The standard deviation is the error on the mean. This 
+        method assumes that the sample follows a gaussian 
+        distribution, and works best for large samples. This 
+        method works well to find an estimate of a fixed value 
+        that has been measured multiuple times with some random 
+        error.
+
+    axis: integer or None
+        Only when the sample is a numpy array. The axis along 
+        which the ufloats are computed. If None (the default value)
+        the sample is the whole flattened array. 
+    
+    '''
+    
+    if method=="gaussian":
+
+        if numpy is None:
+            #if numpy is not present, use pythons statistics functions instead
+            mean_value=stats_mean(sample)
+            error_on_mean=stats_stdev(sample)/sqrt(len(sample)-1)
```

Comment:
I believe the `statistics.stdev` already divides by `sqrt(len(samples)-1)`, and it should not be done again here.

But also, I think we will need to discuss/document this, and perhaps have an option for using the "-1" ("Bessel's -1"), perhaps adding a `ddof` argument (default=0).   I think the default should be "match numpy.std()", which divides variance by `sqrt(len(samples))`.

Reply:
Yes, you are right, i mistakenly assumed that `statistics.stdev` behaved the same as `numpt.std` does without any optional arguments. 

Adding `ddof=0` as an optional parameter should be simple. For the non-numpy code I can use `statistics.pstdev` and then include "-ddof" in the denominator.